src/ctapipe/io/__init__.py

```diff
@@ -22,32 +22,45 @@
     DL1_GROUP,
```

Comment:
I think it is fine to not expose these values at the `ctapipe.io` toplevel init.

It's fine to have them imported from `ctapipe.io.hdf5dataformat` or just use `from ctapipe.io import hdf5dataformat` and use `hdf5dataformat.<some member>`

Reply:
I think this is very low-level info to be exposed at the top-level io module. So yes, I'd prefer to remove here I think.

Reply:
Done